GooglePlayInstant/Editor/QuickDeploy/AssetBundleVerifierWindow.cs

```diff
@@ -54,7 +54,7 @@ private void StartAssetBundleVerificationDownload()
             _webRequest.SendWebRequest();
         }
 
-        //TODO: Support Unity 5.6.0+
+        //TODO: Support Unity 5.6.0+ and fix random crashing
```

Comment:
this sounds ominous, let's at least add some context on what is crashing

Reply:
At the time of writing this, I hadn't known the source of the issue; could not reproduce. Audace helped me clear some of this. Resolved.